utils/merge-rulesets.py

```diff
@@ -81,6 +81,9 @@ def normalize(f):
 		elif child.tag == "exclusion":
 			ruleset["exclusion"].append(child.attrib["pattern"])
 
+	if "exclusion" in ruleset:
```

Comment:
It looks we  overwrite the `ruleset["exclusion"]` that we just set above? Is this correct?

Reply:
Yes, but outside the loop.